src/PublicApiGeneratorTests/Interface_methods.cs

```diff
@@ -93,7 +93,7 @@ public interface IMethodsWithParameters
 
         public interface IMethodsWithDefaultParameters
         {
-            void Method1(int intValue = 42, string stringValue = "hello world", Type typeValue = null);
+            void Method1(int intValue = 42, string stringValue = "hello world", Type typeValue = null!);
```

Comment:
In real code, this should be `Type?`. I didn't want to fuss with changing the tests.

Another way to handle this could be to do `#nullable disable` in each test file for the test types section.